pkg/sql/catalog/tabledesc/validate_test.go

```diff
@@ -1977,7 +1977,7 @@ func TestValidateTableDesc(t *testing.T) {
 	}
 	for i, d := range testData {
 		t.Run(d.err, func(t *testing.T) {
-			d.desc.Privileges = catpb.NewBasePrivilegeDescriptor(username.RootUserName())
+			d.desc.Privileges = catpb.NewReadWriteSystemTablePrivilegeDescriptor(username.RootUserName())
```

Comment:
Should we update these tables to not be part of the system table and create a separate test for system tables here?

Reply:
I did start on that path but the diff was a lot larger so I switched to this more minimal diff. I figured the Privileges we were setting here weren't the crux of what we were testing so it didn't matter much. I'm happy to go through the motions of bumping all tableIDs to 2 and their parents to 3 or something if others feel strongly about it.

Reply:
This is moot now that we only special case for the system tables created by the restore to copy rows over.